src/toxcore/dht/packet/mod.rs

```diff
@@ -17,7 +17,7 @@ mod lan_discovery;
 mod crypto_handshake;
 mod crypto_data;
 mod cookie;
-mod errors;
+pub mod errors;
```

Comment:
Why `pub mod` when you have `pub use` next?

Reply:
https://github.com/tox-rs/tox/blob/25db53d1eb30b2e0a49d4b947a2f3138d5770ce1/src/toxcore/onion/packet/onion_announce_request.rs#L8

errors.rs is used by onion module.